From e9bacf8ea7e145d2edb2b29a1aeb2be7624b6424 Mon Sep 17 00:00:00 2001
From: Nikita Travkin <nikitos.tr@gmail.com>
Date: Wed, 5 May 2021 14:21:14 +0500
Subject: [PATCH 01/24] Revert "FIXME: arm64: dts: qcom:
 apq8016-samsung-gt510wifi: Add audio jack detection"

This reverts commit 85ad9b900e03f3766b10eb898b8bb70205be0295.
---
 .../dts/qcom/apq8016-samsung-gt510wifi.dts     | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/apq8016-samsung-gt510wifi.dts b/arch/arm64/boot/dts/qcom/apq8016-samsung-gt510wifi.dts
index 7982317fe3..9f8fac8af5 100644
--- a/arch/arm64/boot/dts/qcom/apq8016-samsung-gt510wifi.dts
+++ b/arch/arm64/boot/dts/qcom/apq8016-samsung-gt510wifi.dts
@@ -276,16 +276,6 @@ &usb_hs_phy {
 	extcon = <&usb_vbus>;
 };
 
-&wcd_codec {
-	jack-gpios = <&msmgpio 110 GPIO_ACTIVE_LOW>;
-	qcom,micbias-lvl = <2800>;
-	qcom,mbhc-vtreshold-low = <75 150 237 450 500>;
-	qcom,mbhc-vtreshold-high = <75 150 237 450 500>;
-
-	pinctrl-names = "default";
-	pinctrl-0 = <&jack_default>;
-};
-
 &smd_rpm_regulators {
 	vdd_l1_l2_l3-supply = <&pm8916_s3>;
 	vdd_l4_l5_l6-supply = <&pm8916_s4>;
@@ -414,14 +404,6 @@ gpio_hall_sensor_default: gpio-hall-sensor-default {
 		bias-disable;
 	};
 
-	jack_default: jack-default {
-		pins = "gpio110";
-		function = "gpio";
-
-		drive-strength = <2>;
-		bias-disable;
-	};
-
 	lcd_en_default: lcd-en-default {
 		pins = "gpio8";
 		function = "gpio";
-- 
2.25.1

